android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/Addon.kt

```diff
@@ -44,7 +44,7 @@ val logger = Logger("Addon")
  * @property installedState Holds the state of the installed web extension for this add-on. Null, if
  * the [Addon] is not installed.
  * @property defaultLocale Indicates which locale will be always available to display translatable fields.
- * @property reviewUrl The link to the review page for this [Addon].
+ * @property ratingUrl The link to the ratings page (user reviews) for this [Addon].
```

Comment:
So... it's called `ratings_url` on AMO (API) but `reviewURL` in Gecko and, in this project, we assign `ratings` (the object) to `rating` (`Rating` class). This is why this property ends up being `ratingUrl`.